### ISSUE TITLE
epochs setting

### ISSUE THREAD

**Issue by Donghyun-Kang-Soc** [css-research/hw01#2]
Hi!
I found that question 3-(i) states "Identify the epoch where the model's performance degrades based on the validation set."; I'm wondering the number of epochs for 3-(ii) and 3-(iii) should be adjusted based on the result from the initial test...

**Comment by bensoltoff** [comment]
You can extend the number of epochs if the validation error doesn't stabilize after the initial run, though I think it should? I'll check my results when I get to the office today

**Comment by bensoltoff** [comment]
Hmm, after a second check you may want to extend the number of epochs for a couple of the model formulations. Due to the slower learning imposed by regularization/dropout, you may need to train the model longer to achieve improved performance over the base model.

**Comment by Donghyun-Kang-Soc** [comment]
I'll try it thanks.
